tests/x11/yast2_lan_restart.pm

```diff
@@ -228,7 +228,9 @@ sub run {
     become_root;
     # make sure that firewalld is stopped, or we have later pops for firewall activation warning
     # or timeout for command 'ip a' later
-    if (script_run "systemctl show -p ActiveState firewalld.service | grep ActiveState=active") {
+    if (((is_sle && sle_version_at_least('15')) || (is_leap && leap_version_at_least('15.0')))
```

Comment:
leap_version_at_least returns if it's not leap, so 'is_leap' is not required. 
As for is_sle && sle_version_at_least('15') we have 22 usages of it, so I guess it's time to extract it to version_utils.

Reply:
In general yes but I would rather change the function `sle_version_at_least` to only match on sle so that we can replace all occurences and be consistent with `leap_version_at_least`. I keep forgetting to create a progress ticket about that.